sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -61,7 +61,7 @@ public final class StorageBlobClient: PipelineClient {
         "https://storage.azure.com/.default"
     ]
 
-    private static let manager: TransferManager = URLSessionTransferManager.shared
+    private static var manager: TransferManager = URLSessionTransferManager.shared
```

Comment:
Any particular reason for this change?

Reply:
It was to allow you to set the concurrent operations on the TM. I modified it so this isn't needed.